README.md

```diff
@@ -168,13 +168,14 @@ rm-ds-store
 
 ### 🖼 Show certificate information of a service
 
-Script: [show-certificate](show-certificate)  
-Dependencies: [openssl](https://www.openssl.org/)
+Script: [show-certificate](show-certificate)
 
 ```sh
 show-certificate google.com
 ```
 
+Depends on openssl.
```

Comment:
Please revert this change with dependencies as that's how dependencies are listed everywhere.

Reply:
Done